google/cloud/bigquery/job/query.py

```diff
@@ -674,6 +674,21 @@ def write_disposition(self):
     def write_disposition(self, value):
         self._set_sub_prop("writeDisposition", value)
 
+    @property
+    def write_incremental_results(self) -> Optional[bool]:
+        """This is only supported for a SELECT query using a temporary table. 
+        
+        If set, the query is allowed to write results incrementally to the temporary result 
+        table. This may incur a performance penalty. This option cannot be used with Legacy SQL.
+
+        This feature is not generally available.
+        """
+        return self._get_sub_prop("writeIncrementalResults")
```

Comment:
I assume this isn't allowed for the low latency query path? But if it is, you can add it here to the allow list: https://github.com/googleapis/python-bigquery/blob/main/google/cloud/bigquery/_job_helpers.py#L539

Reply:
Plumbed this through an added a test.